src/cutadapt/adapters.py

```diff
@@ -654,6 +683,11 @@ def _aligner(self) -> Aligner:
             Where.ANYWHERE.value if self._force_anywhere else Where.FRONT.value
```

Comment:
To fix #665, this was changed to `... else Where.BACK.value` in main, so `_kmer_finder` should also be adjusted accordingly.

Reply:
Ah sorry, wrong class.